src/parser/shaman/restoration/modules/core/HealingRainLocation.js

```diff
@@ -24,9 +22,10 @@ class HealingRainLocation extends Analyzer {
 
   constructor(...args) {
     super(...args);
-    const hasDeluge = this.selectedCombatant.hasTalent(SPELLS.DELUGE_TALENT.id);
-    const hasRebalancers = this.selectedCombatant.hasFeet(ITEMS.ELEMENTAL_REBALANCERS.id);
-    this.active = hasDeluge || hasRebalancers;
+    this.active = this.selectedCombatant.hasTalent(SPELLS.DELUGE_TALENT.id);
+    if (this.active && this.selectedCombatant.hasTrait(SPELLS.OVERFLOWING_SHORES_TRAIT.id)) {
+      this.healingRainDiameter *= Math.pow(1.1,this.selectedCombatant.traitRanks(SPELLS.OVERFLOWING_SHORES_TRAIT.id).length);
```

Comment:
I don't know of any trait effects that don't scale with ilvl that also stack. This one doesn't scale with ilvl, are you sure it stacks?

Reply:
yeah, i tested it extensively. each stack has a very noticeable increase in size. here it is 0 -> 3 stacks which is far more than 10%
![image](https://user-images.githubusercontent.com/2842471/47180788-adea6000-d321-11e8-96d6-ad5b6049901f.png)

Reply:
wow, good to know!

I should probably go retest a couple of traits to triple-check that they don't stack, sigh...